javascript/code-challenges/challenges-02.test.js

```diff
@@ -0,0 +1,307 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+
+CHALLENGE 1 - Review
+
+Write a function named raisedToTheThird that takes in an array of numbers and returns a new array of each of those numbers raised to the 3rd power (hint: look up Math.pow()). Use forEach to solve this problem.
+
+------------------------------------------------------------------------------------------------ */
+
+const raisedToTheThird = (arr) => {
+  // Solution code here...
+  let arr2 = [];
+  let newNum = arr.map(num => Math.pow(arr, 3));
+  arr2.push(newNum);
+
+
+  // arr.forEach(element => {
+  //   let newNum = element + 1;
+  //   arr2.push(newNum);
+  // });
+  return arr2;
+};
+
```

Comment:
Hi there, good work on these so far! (I had accidentally looked at the wrong challenge previously, if you get an email you can disregard :)). I believe we need to make a new branch for challenges each day (see assignment instructions). 👍 

- For this one, we should use `forEach`
-  For each element in the input arr, we can use Math.pow(like you are above). 
- You already have `arr2` defined, so you're still on the right track with your push() as well.
- `arr2` is our return variable, we might consider changing this variable name to be something descriptive (it raises all nums to third power, so maybe `thirdPowerNums` etc.